pkg/interface/chat/src/js/components/member.js

```diff
@@ -69,56 +72,86 @@ export class MemberScreen extends Component {
       );
     });
 
+      let popoutSwitcher = this.props.popout
+        ? "dn-m dn-l dn-xl"
+        : "dib-m dib-l dib-xl";
+      let isinPopout = this.props.popout ? "popout/" : "";
+
     return (
       <div className="h-100 w-100 overflow-x-hidden flex flex-column">
-        <div className='pl3 pt2 bb mb3'>
-          <h2>{state.station.substr(1)}</h2>
+        <div className="w-100 dn-m dn-l dn-xl inter pt4 pb6 pl3 f8"
+             style={{ height: "1rem" }}>
+          <Link to="/~chat/">{"⟵ All Chats"}</Link>
+        </div>
+        <div className="pl3 pt4 bb b--gray4 flex relative overflow-x-scroll flex-shrink-0"
+             style={{ height: 48 }}>
+          <a className="pointer"
+            onClick={() => {
+              store.setState(previousState => ({
+                sidebarShown: !previousState.sidebarShown
```

Comment:
Don't call store setState manually. Call it via a fake API action.

Reply:
Added